website/blog/2025-01-28-git-branching-strategies-and-dbt.md

```diff
@@ -374,32 +375,32 @@ We highly recommend that you choose your branching strategy based on which *best
 
 - **Indirect promotion**
     
     Strengths
     
     - There’s a dedicated environment to test end-to-end changes over time.
-    - Data output can be reviewed either with a developer on PR or once things hit the middle branch.
-    - Review from other tools is much easier, because the middle branch tends to deploy to a centralized location. “Staging” reports can be set up to always refer to this location for reviewing changes, and processes for creating new reports can flow from staging to production.
+    - Data outputs can be reviewed either with a developer on PR or once things are in the middle branch.
+    - Review from other tools is much easier because we have the option of deploying our middle branch to a centralized location. “Staging” reports can be set up to always refer to this location for reviewing changes, and processes for creating new reports can flow from staging to production.
     - Configurations and job changes can be tested with production-like parameters before they actually hit production.
-    - There’s a dedicated environment to merge changes if you need them for shared development. Consumers of `main` will be none-the-wiser about the things that developers do for ease of collaboration.
+    - Changes merged to the middle branch for shared development won't be reflected in production. Consumers of `main` will be none-the-wiser about the things that developers do for ease of collaboration.
     
     Weaknesses
     
     - Changes can be slower to get to production due to the extra processes intended for the middle branch. In order to keep things moving, there should be someone (or a group of people) in place who fully own managing the changes, validation status, and release cycle.
     - Changes that are valid can get stuck behind other changes that aren’t - having a good plan in place for how the team should handle this scenario is essential because conundrum can hold up getting things to production.
-    - There’s extra management of any new trunks, which will need ownership - without someone (or a group of people) who are knowledgeable, it can be confusing understanding what needs to be done how to do it when things get out of sync.
-    - Requires additional compute in the form of scheduled jobs in the qa environment as well as an additional CI job from qa > main
+    - There’s extra management of any new trunks, which will need ownership - without someone (or a group of people) who are knowledgeable, it can be confusing understanding what needs to be done and how to do it when things get out of sync.
+    - It can require additional compute in the form of scheduled jobs in the QA environment, as well as an additional CI job from qa > main for testing releases before they're merged.
```

Comment:
```suggestion
    - It can require additional compute in the form of scheduled jobs in the QA environment, as well as an additional CI job from `qa` > `main` for testing releases before they're merged.
```